Webagent (#10)
airtop-ai/examples-python: examples/web_agent/How to schedule worker.md

> # How to run automatically the worker script

Reviewer: ```suggestion
# How to schedule the worker script
```

Author: Done

In case the site you're acessing requires login, we suggest you hardcode the profile_id in the worker so it runs automatically.
Just edit this line in worker.py

`profile_id = input("Enter a profileId (or press Enter to skip): ").strip()`

# Crontab (Linux, Mac)

To schedule the worker with crontab you can create a cronjob with

`crontab -e`

Example of how to make the worker run every day

`@daily python [PATH TO web_agent]/worker.py`

Full documentation

> https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules?hl=pt-br

Reviewer: It's linking to the Portuguese version. Let's replace with the english version: https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules

Author: Done

# Task Scheduler (Windows)

You can use task scheduler in Windows

https://learn.microsoft.com/en-us/windows/win32/taskschd/daily-trigger-example--scripting-

> # Supervisord

Reviewer: I didn't understand this section. How does it differ from the `Crontab (Linux, Mac)` and the `Task Scheduler (Windows)` sections? When would the user choose to follow these steps?

Author: I think it's just a matter of different services that can do the scheduling. I've worked with companies that chose one of them. Supervisord is not dependant on the OS. I just thought I would put everything I knew of how to schedule

Reviewer: ok. I rewrote the description to something that feels like this is just another option to schedule a task

> You can use supervisord to setup the automation of your task

Reviewer: ```suggestion
You can also use supervisord as a task scheduler
```

Author: Done
